### DISCUSSION TITLE
speed up the solution process

### DISCUSSION THREAD

**Discussion by ellensonny** [yalmip/YALMIP#1421] [General discussion]
Dear,Prof:
When I do not give an objective function, the solution process is as follows：
![image](https://github.com/yalmip/YALMIP/assets/130203697/ae8a662e-cc69-4b63-bd52-c1ac03528ed4)
![image](https://github.com/yalmip/YALMIP/assets/130203697/e5595133-a31d-4877-b85a-e18d52473e73)

which takes about 60 seconds in total.
However When I give an objective function as follow in workspace:
![image](https://github.com/yalmip/YALMIP/assets/130203697/eb8c61ba-e4e9-490a-bdc9-6138235ec971)
 the solution process takes a particularly very long time.Here is the process:
![image](https://github.com/yalmip/YALMIP/assets/130203697/5783ba40-28c7-48c6-9004-43a8124dbae2)
Could you please provide valuable advice to speed up the solution process? Thank you very much for your help.

**Comment by johanlofberg** [comment]
There is no simple way without specific model information. You have a mixed-integer nonconvex quadratic problem, so in theory the problem is simply not solvable. In practice you can be lucky (which you were for the feasibility problem but not when searching for an optimal solution)

One things is clear from the log though, the objective is extremely badly scaled as the root relaxation has an objective of 10^8 (and gurobi also warns about nasty numerics in objective)

**Reply by ellensonny** [reply]
Maybe I should describe the model in a better way. I will try again.thank you for your reply
